File: Janari0.Model/Order.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Janari0.Model
{
    public class Order
    {
        public int OrderId { get; set; }

        public string OrderNumber { get; set; }

        public DateTime Date { get; set; }

        public bool Status { get; set; }

        public bool Canceled { get; set; }

        public int BuyerId { get; set; }

        public float Price { get; set; }

        public virtual Buyer Buyer { get; set; }

        public virtual ICollection<OrderItem> OrderItems { get; set; } = new List<OrderItem>();

        public virtual ICollection<Output> Outputs { get; set; } = new List<Output>();
    }
}

File: janari0_mobile/lib/model/order.dart
import 'package:json_annotation/json_annotation.dart';

import 'orderItem.dart';

part 'order.g.dart';

@JsonSerializable(explicitToJson: true)
class Order {
  int? orderId;
  String? orderNumber;
  int? buyerId;
  String? date;
  bool? status;
  bool? canceled;
  double? price;
  List<OrderItem>? orderItems;

  Order();

  factory Order.fromJson(Map<String, dynamic> json) => _$OrderFromJson(json);

  /// Connect the generated [_$OrderToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$OrderToJson(this);
}
